File: domain/src/main/java/com/aceplus/domain/model/forApi/Township.java
package com.aceplus.domain.model.forApi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by phonelin on 2/13/17.
 */
public class Township {

    @SerializedName("township_id")
    @Expose
    private String townshipId;
    @SerializedName("township_name")
    @Expose
    private String townshipName;

    public String getTownshipId() {
        return townshipId;
    }

    public void setTownshipId(String townshipId) {
        this.townshipId = townshipId;
    }

    public String getTownshipName() {
        return townshipName;
    }

    public void setTownshipName(String townshipName) {
        this.townshipName = townshipName;
    }


}

File: domain/src/main/java/com/aceplus/domain/entity/predefine/Township.kt
package com.aceplus.domain.entity.predefine

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import android.arch.persistence.room.Query
import com.aceplus.domain.model.route.Route_Township
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "township")
class Township {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    @Expose
    var id: Int = 0

    @ColumnInfo(name = "township_id")
    @SerializedName("township_id")
    @Expose
    var township_id:  String? = null

    @ColumnInfo(name = "township_name")
    @SerializedName("township_name")
    @Expose
    var township_name: String? = null


}